x-pack/plugins/security_solution/cypress/screens/rule_details.ts

```diff
@@ -104,3 +104,9 @@ export const removeExternalLinkText = (str: string) =>
   str.replace(/\(opens in a new tab or window\)/g, '');
 
 export const BACK_TO_RULES = '[data-test-subj="ruleDetailsBackToAllRules"]';
+
+export const hasIndexPatterns = (indexPatterns: string) => {
+  cy.get(DEFINITION_DETAILS).within(() => {
+    getDetails(INDEX_PATTERNS_DETAILS).should('have.text', indexPatterns);
+  });
+};
```

Comment:
Looks like a task that should be placed in the `tasks` folder maybe? At least based on our current approach to structuring Cypress code.

Reply:
you are right, I've got confused by already existing `getDetails` within this file. So, I moved it as well to tasks.